pkg/ipfs/node.go

```diff
@@ -431,7 +431,15 @@ func createRepo(path string, nodeConfig types.IpfsConfig) error {
 
 // loadPlugins initializes and injects the standard set of ipfs plugins.
 func loadPlugins(cm *system.CleanupManager) error {
-	plugins, err := loader.NewPluginLoader("")
+	// We use a temporary folder for the plugin loader so that when it
+	// does the dynamic plugin loading (which cannot be turned off) it
+	// does not break when it finds a binary in a local ./plugins folder
+	repoDir, err := os.MkdirTemp("", "")
```

Comment:
Unfortunately it tries to walk the directory. We already run the config for devstack in a temp folder so that we don't mangle the stuff that's in ~/.bacalhau